plexus-compiler-its/pom.xml

```diff
@@ -39,7 +39,8 @@
               <goal>verify</goal>
             </goals>
             <configuration>
-              <debug>true</debug>
+              <debug>false</debug>
```

Comment:
When searching for the problem in the IT, I noticed that the debug output was confusing me more than helping me. I think, it is better to activate it on a case-by-case basis. The standard build also does not run in Maven debug mode.